### PR TITLE
Update limit for trace filter

### LINKED ISSUE #1487: The `trace-filter` method in JSON-RPC API now has a default block range limit of 1000
The `trace-filter` method in JSON-RPC API now has a default block range limit of 1000, adjustable with `--rpc-max-trace-filter-range`

See: https://github.com/hyperledger/besu/pull/6446

### PR DESCRIPTION
**Pull request by joaniefromtheblock**
Closes #1487

### CODE REVIEW

**Review by Gabriel-Trintinalia — COMMENTED** [review]
Lgtm

**Review by mjsmike62 — APPROVED** [review]
LGTM

File: docs/public-networks/reference/api/index.md
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:6970]:
```suggestion
```

Why these added line breaks?

**Review by alexandratran — CHANGES_REQUESTED** [review]
The `--rpc-max-trace-filter-range` option is a newly added command line option (you can examine the [original product PR](https://github.com/hyperledger/besu/pull/6446/files) to see that it's been added to the source code). This means that you need to add documentation for it to the [Besu command line options reference](https://besu.hyperledger.org/public-networks/reference/cli/options) in the appropriate location on the page (after `rpc-max-logs-range`, since the options are alphabetized).

File: docs/public-networks/reference/api/index.md
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:7066]: I'm not sure this needs to be mentioned here, but if it does, please move it up into the description for this method ("Returns traces matching the specified filter.") instead of after the example.

File: docs/public-networks/reference/api/index.md
Comment (alexandratran) [inline docs/public-networks/reference/api/index.md:6963]:
```suggestion
Returns traces matching the specified filter. The maximum number of blocks you can supply to `trace_filter` is 1000 by default. You can adjust this limit using the [`--rpc-max-trace-filter-range`](../cli/options.md#rpc-max-trace-filter-range) option. 
```

File: docs/public-networks/reference/cli/options.md
Comment (alexandratran) [inline docs/public-networks/reference/cli/options.md:4049]:
```suggestion
The maximum number of blocks you can supply to the [`trace_filter`](../api/index.md#trace_filter) method. The value must be equal to or greater than `0`. Setting this option to `0` indicates there is no limit. The default is `1000`.
```

Make sure the option description begins similarly to other option descriptions on this page. Always provide a default value if it exists.

**Review by alexandratran — COMMENTED** [review]
Looks a lot better, I added some minor suggestions for clarity around this topic. Will approve once those are addressed.

**Comment by joaniefromtheblock** [comment]
Thanks @alexandratran; I added your suggestions. I think I just need approval from Eng.

**Review by alexandratran — APPROVED** [review]

**Pull request merged**
